src/runmode-af-xdp.c

```diff
@@ -368,7 +408,7 @@ int RunModeIdsAFXDPWorkers(void)
     RunModeInitialize();
     TimeModeSetLive();
 
-    (void)ConfGet("af-xdp.live-interface", &live_dev);
+    ret = ConfGet("af-xdp.live-interface", &live_dev);
```

Comment:
I'm not sure I understand why to get this return value here, if we won't use it.